specification/network/resource-manager/Microsoft.Network/2017-08-01/usage.json

```diff
@@ -92,6 +92,11 @@
     },
     "Usage": {
       "properties": {
+        "id": {
```

Comment:
This will be a breaking change to a generated class.

Reply:
it is a read only string right? why is that considered a breaking change? what do you suggest we do?

this is a bug fix we are trying to do

Reply:
If it's a bugfix, I'm not going to flag it-- it's about as harmless as can be, when introducing a new property into a class where it wasn't before. Likely it's not going to screw anything up.